pkg/html/html.go

```diff
@@ -292,7 +291,18 @@ func summaryJobsByPlatform(report, reportPrev util.TestReport, endDay, jobTestCo
 
 			encodedTestName := url.QueryEscape(regexp.QuoteMeta(test.Name))
 
-			rows = rows + fmt.Sprintf(testGroupTemplate, strings.ReplaceAll(v.Platform, ".", ""), test.Name, v.Platform, report.Release, encodedTestName,
+			// Multi-stage jobs that use variants will have the OCP release
+			// before the name of the job, while the older template based
+			// tests will have it after.
```

Comment:
can we do a single regex which is `release.platform | platform.release` and do away w/ the special case for metal-ipi?

Reply:
Oh yea, makes sense. Updated!